File: app/src/main/java/com/kelsos/mbrc/events/ui/NotifyUser.java
package com.kelsos.mbrc.events.ui;

public class NotifyUser {
  private String message;
  private int resId;
  private boolean fromResource;

  public NotifyUser(String message) {
    this.message = message;
    this.fromResource = false;
  }

  public NotifyUser(int resId) {
    this.resId = resId;
    this.fromResource = true;
  }

  public String getMessage() {
    return message;
  }

  public int getResId() {
    return resId;
  }

  public boolean isFromResource() {
    return fromResource;
  }
}

File: app/src/main/java/com/kelsos/mbrc/events/ui/NotifyUser.kt
package com.kelsos.mbrc.events.ui

class NotifyUser {
  val message: String
  val resId: Int
  var isFromResource: Boolean = false
    private set

  constructor(message: String) {
    this.message = message
    this.isFromResource = false
    this.resId = -1
  }

  constructor(resId: Int) {
    this.resId = resId
    this.isFromResource = true
    this.message = ""
  }
}
